client/src/components/templates/AlumniLayout/index.tsx

```diff
@@ -14,6 +16,7 @@ type Props = {
 
 const AlumniLayout: FC<Props> = ({ metaTitle, children }): JSX.Element => {
   const { data: alumni, error } = userHooks()
+  const { alumniDataList } = alumniHooks()
   const [isOpen, setIsOpen] = useState<boolean>(false)
```

Comment:
Normally, the naming convention for create a custom hook is to pre-pend `use` on to the function.
Sample: `useAlumni`